### PR TITLE
tox/pytest: support posargs, move to testpaths

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -9,7 +9,7 @@ license_file = LICENSE
 
 [tool:pytest]
 minversion = 3.0
-testpaths = tests
+testpaths = tests examples
```
Comment (davidism) [inline setup.cfg]: The reason this is set in tox is because I want people to be able to run `pytest` without having to install the examples.

Comment (blueyed) [reply setup.cfg]: I see.. `{posargs:tests examples}` in tox.ini then?

Comment (blueyed) [reply setup.cfg]: updated

**Comment by davidism** [comment]
Ended up addressing this while moving to Azure. Thanks.

**Pull request closed without merging**